conda_build/skeletons/cran.py

```diff
@@ -620,8 +661,10 @@ def make_array(m, key, allow_empty=False):
     return result
 
 
-def existing_recipe_dir(output_dir, output_suffix, package):
+def existing_recipe_dir(output_dir, output_suffix, package, version):
     result = None
+    if version:
```

Comment:
When a version is explicitly requested—even if it happens to be the latest available version—we append that to the directory name. I'm using this now because I'm building multiple versions of the same package against the same base R version for a customer.